js/src/dapps/registry/accounts/accounts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+
+// import styles from './lookup.css';
+
+const renderAccount = (active) => (account) => (
```

Comment:
Makes sense.

I'd be careful however with changing code in favor of the inner workings of React. As long as you expect dom diffing to be fast, this won't be a problem and less specific to React.